realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -411,6 +411,7 @@ private static void initializeSyncedRealm(Realm realm) {
                 realm.setVersion(newVersion);
                 commitChanges = true;
             }
+            schema.close();
```

Comment:
If we need `schema.close()` here, we also need to close it at line 405.
Shouldn't we use `try - finally`?